### PR TITLE
Initial commit for front end local

### PR DESCRIPTION
**Pull request by josharnoldjosh**
I created some front-end code for the applications framework.

### CODE REVIEW

File: applications-frontend/src/sass/styles.scss [hunk 1]
```diff
@@ -14,4 +49,5 @@ $navbar-item-img-max-height: 3rem;
 // 6. Import the rest of Bulma
 @import "~bulma";
 
-@import "flex";
\ No newline at end of file
+@import "flex";
```
Comment (thebearjew) [inline applications-frontend/src/sass/styles.scss]: I would import these first.

CSS is cascading so anything you define above this import could be overwritten by `bulma` if they share a common selector.

You probably want to import bulma and then add custom styles.

File: applications-frontend/src/components/Applicant.vue [hunk 2]
```diff
@@ -1,61 +1,83 @@
...
+		<!-- Control panel -->
+		<div class="column has-background-light is-3">		
+			<center>
+				<!-- Person's Image -->
+				<figure class="image is-128x128">
+					<img class="profile-picture" alt="" src="https://bulma.io/images/placeholders/256x256.png">
+				</figure>
+
+				<!-- Person's name -->			
+				<div class="subtitle is-4" style="margin: 16px;">Full-name</div>
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: bind to a value in data()

File: applications-frontend/src/components/Applicant.vue [hunk 3]
```diff
@@ -1,61 +1,83 @@
...
+
+				<!-- Break for looking good :D -->
+				<div class="pad-light"></div>
+				<div class="break"></div>
+				<div class="pad-light"></div>
+
+				<!-- Rating control -->
+				<div class="subtitle is-4" style="margin: 16px;">Rating</div>
+				<div class="buttons has-addons is-centered">
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: consider using a set of radio buttons and restyling them, this makes getting the value easier. Right now you will need a callback for this set of buttons

File: applications-frontend/src/components/Applicant.vue [hunk 4]
```diff
@@ -1,61 +1,83 @@
...
+					<span class="button">1</span>
+					<span class="button">2</span>
+					<span class="button">3</span>
+					<span class="button">4</span>
+					<span class="button">5</span>
+				</div>
+
+				<!-- Next applicant button -->
+				<a class="button is-primary is-medium">Next applicant</a>
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: bind onClick to a callback

File: applications-frontend/src/components/Applicant.vue [hunk 5]
```diff
@@ -1,61 +1,83 @@
...
+			<div>
+				<a class="button is-primary is-medium" style="margin-left: 16px;">Resume</a>					
+			</div>			
+			<iframe src="http://josharnoldjosh.github.io" width="80%" height="250px" style="overflow:auto; margin-top: 16px;"></iframe>
+			<div class="pad"></div>
+
+			<!-- Github -->
+			<div class="title" style="margin: 16px;">Github</div>	
+			<a class="button is-primary is-medium" style="margin-left: 16px;">Github</a>
+			<div class="pad"></div>
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: where's the embed we were looking at?

File: applications-frontend/src/components/Applicant.vue [hunk 6]
```diff
@@ -1,61 +1,83 @@
...
+			<div class="pad-light"></div>		
+
+			<!-- Year -->
+			<div class="title" style="margin: 16px;">College year</div>	
+			<div class="subtitle is-4" style="margin: 16px;">{{year}}</div>	
+			<div class="pad-light"></div>	
+
+			<!-- Essays -->
+			<div class="title" style="margin: 16px;">Text information</div>	
+			<div class="subtitle is-4" style="margin: 16px;">REPLACE ME</div>
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: You're going to have to use v-for to insert a series of essay responses here, in a future commit

File: applications-frontend/src/sass/styles.scss [hunk 7]
```diff
@@ -4,6 +4,41 @@
...
+}
+
+.column {
+	margin: 8px;
+	border-radius: 8px;
+}
+
+.profile-picture {	
+    border-radius: 50%;
+    margin-top: 32px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: don't use px, ems

File: applications-frontend/src/sass/styles.scss [hunk 8]
```diff
@@ -4,6 +4,41 @@
...
 // 2. Set your own initial variables
 $navbar-item-img-max-height: 3rem;
+
+.columns {
+	border-radius: 64px;
+	padding: 8px;
+}
+
+.column {
+	margin: 8px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: em

File: applications-frontend/src/sass/styles.scss [hunk 9]
```diff
@@ -4,6 +4,41 @@
 
 // 2. Set your own initial variables
 $navbar-item-img-max-height: 3rem;
+
+.columns {
+	border-radius: 64px;
+	padding: 8px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: em

File: applications-frontend/src/sass/styles.scss [hunk 10]
```diff
@@ -4,6 +4,41 @@
...
+
+.break {
+    display: block;    
+    height: 2px;
+    border: 0;
+    border-top: 1px solid #ccc;
+    margin: 1em 0;    
+    color: black;
+    margin-right: 32px;
+    margin-left: 32px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: margins should be em

File: applications-frontend/src/components/Applicant.vue [hunk 11]
```diff
@@ -1,61 +1,83 @@
...
-    </div>
+<div class="container">
+	<div class="columns">
+
+		<!-- Control panel -->
+		<div class="column has-background-light is-3">		
+			<center>
+				<!-- Person's Image -->
+				<figure class="image is-128x128">
+					<img class="profile-picture" alt="" src="https://bulma.io/images/placeholders/256x256.png">
```
[the remark this answers is not in the document]
Comment (thomasmunduchira) [reply applications-frontend/src/components/Applicant.vue]: I'm collecting profile photo links from Google :)

[the remark this answers is not in the document]
Comment (thomasmunduchira) [reply applications-frontend/src/components/Applicant.vue]: Correction: I got applicants and directors mixed up. I'm collecting pictures of the directors, not of the applicants. What Kaelan said still holds

File: applications-frontend/src/sass/styles.scss [hunk 12]
```diff
@@ -1,53 +1,134 @@
...
 }
-
 .pad-light {
-    padding-top: 4px;
+    padding-top: 2em;
 }
 
-// 3. Set the derived variables
+.control {
+    font-size: 25px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: rem

File: applications-frontend/src/sass/styles.scss [hunk 13]
```diff
@@ -1,53 +1,134 @@
...
+    opacity: 0;
+}
 
-// 5. Add new color variables to the color map.
-@import "~bulma/sass/utilities/derived-variables";
+.control__indicator {
+    position: absolute;
+    top: 4px;
+    left: 0;
+    width: 30px;
```
Comment (klanmiko) [inline applications-frontend/src/sass/styles.scss]: use em

**Comment by josharnoldjosh** [comment]
Please review latest commit!

File: applications-frontend/src/components/Applicant.vue [hunk 14]
```diff
@@ -1,61 +1,123 @@
...
+				
+				<!-- Next applicant button -->
+				<a v-on:click="next_applicant" class="button is-primary is-medium" style="margin-top: 1em;">Next applicant</a>				
+			</center>		
+		</div>
+
+		<!-- Information panel -->
+		<div class="column has-background-light" style="border-radius: 1em;">
+
+			<div class="columns is-centered is-multiline">
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: two things: can we add a h2 that says "information", and these shouldn't all be their own column, can you make just two columns and then just add the data to each? That would be better

**Pull request closed without merging**